Source/Libraries/Adapters/FileAdapters/ProcessLauncher.cs

```diff
@@ -38,953 +38,902 @@
-            }
+            m_processUtilizationCalculator.Dispose();
         }
-
-        /// <summary>
-        /// Initializes <see cref="ProcessLauncher"/>.
-        /// </summary>
-        public override void Initialize()
+        finally
         {
-            base.Initialize();
+            m_disposed = true;          // Prevent duplicate dispose.
+            base.Dispose(disposing);    // Call base class Dispose().
+        }
+    }
 
-            Dictionary<string, string> settings = Settings;
-            ProcessWindowStyle windowStyle;
-            int initialInputProcessingDelay, utilizationCalculationInterval;
-            string setting;
+    /// <summary>
+    /// Initializes <see cref="ProcessLauncher"/>.
+    /// </summary>
+    public override void Initialize()
+    {
+        base.Initialize();
 
-            ProcessStartInfo startInfo = m_process.StartInfo;
+        Dictionary<string, string> settings = Settings;
 
-            // Load required parameters
-            if (settings.TryGetValue(nameof(FileName), out setting))
-            {
-                setting = FilePath.GetAbsolutePath(setting.Trim());
+        ProcessStartInfo startInfo = m_process.StartInfo;
 
-                if (File.Exists(setting))
-                {
-                    FileName = setting;
-                    startInfo.FileName = FileName;
-                }
-                else
-                {
-                    throw new FileNotFoundException($"Cannot launch process: specified executable path and filename \"{setting}\" does not exist.");
-                }
+        // Load required parameters
+        if (settings.TryGetValue(nameof(FileName), out string setting))
+        {
+            setting = FilePath.GetAbsolutePath(setting.Trim());
+
+            if (File.Exists(setting))
+            {
+                FileName = setting;
+                startInfo.FileName = FileName;
             }
             else
             {
-                throw new ArgumentException($"Cannot launch process: required \"{nameof(FileName)}\" parameter is missing from connection string.");
+                throw new FileNotFoundException($"Cannot launch process: specified executable path and filename \"{setting}\" does not exist.");
             }
+        }
+        else
+        {
+            throw new ArgumentException($"Cannot launch process: required \"{nameof(FileName)}\" parameter is missing from connection string.");
+        }
 
-            // Load optional parameters
-            if (settings.TryGetValue(nameof(SupportsTemporalProcessing), out setting))
-                m_supportsTemporalProcessing = setting.ParseBoolean();
-            else
-                m_supportsTemporalProcessing = DefaultSupportsTemporalProcessing;
+        // Load optional parameters
+        m_supportsTemporalProcessing = settings.TryGetValue(nameof(SupportsTemporalProcessing), out setting) && setting.ParseBoolean();
```

Comment:
I get that these are equivalent because `DefaultSupportsTemporalProcessing` is `false`, but it still seems wrong to not use it here.
```suggestion
        m_supportsTemporalProcessing = settings.TryGetValue(nameof(SupportsTemporalProcessing), out setting) ? setting.ParseBoolean() : DefaultSupportsTemporalProcessing;
```